### ISSUE TITLE
BioLQM models should have an associated initial state

### ISSUE THREAD

**Issue by jonrkarr** [GINsim/GINsim-python#19]
Running simulations with synchronous updating produces the same results from SBML files with different initialLevels.

**Comment by aurelien-naldi** [comment]
Yes, it does ignore it as bioLQM models have no internal state (SBML is the only input format specifying an initial level).
It would be nice to support it, at least for simulations, maybe in other places as well.

**Comment by aurelien-naldi** [comment]
As stated in #18, the addition of an initial state to the model itself would require the addition of getter/setters for this new object
